### DISCUSSION TITLE
映像,音声ファイル用の別枠メディアプレイヤーがあると便利?

### DISCUSSION THREAD

**Discussion by ikakonbu** [misskey-dev/misskey#11837] [Q&A] [answered]
### 概要
別ウインドウのメディアプレイヤーがあると便利かも？

### 目的
・画像に比べて動画、音声ファイルの存在感が薄い
・TLの文字情報と並列でメディアの音声を流し聞きしたい
ので、別ウインドウでメディアプレイヤーがあるとクリエイター(&ミス廃)に優しいのでは

### 機能
・添付ファイルのそばに  `キューに追加`  ボタンを作り、ボタンを押すとキューに追加される
・キューに追加されたメディアはポップアップ上のプレーヤーで再生される
・キュー内でループ再生、連続再生、個別の音量調整ができる
・作ったキューは共有できるようにする(プレイリスト的な使い方ができる)

### 提案元
https://misskey.io/notes/9jpsxva48v

**Comment by acid-chicken** [comment, marked as answer]
Duplicate of #85
